<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="FTR3COF93BK8A76" ignored="0" name="test_1" tests="8" failures="2" timestamp="2022-03-24T04:33:03 EDT" time="159.297" errors="0">
  <testcase name="DT_01" time="15.245" classname="TestCases.WFA_POTS_Dial_Tone_1">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [106] line_end; expected [I would like to assist you in determining if the trouble is at your location or in your telephone equipment. Are you interested in hearing the trouble-shooting steps?If the customer is interested, select &amp;apos;Willing to check&amp;apos;] but found [I would like to assist you in determining if the trouble is atyour location or in your telephone equipment. Are youinterested in hearing the trouble-shooting steps?If the customer is interested, select &amp;apos;Willing to check&amp;apos;]gui_start  gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
@@line_start [106] line_end; expected [I would like to assist you in determining if the trouble is at your location or in your telephone equipment. Are you interested in hearing the trouble-shooting steps?If the customer is interested, select 'Willing to check'] but found [I would like to assist you in determining if the trouble is atyour location or in your telephone equipment. Are youinterested in hearing the trouble-shooting steps?If the customer is interested, select 'Willing to check']gui_start  gui_end
at TestCases.WFA_POTS_Dial_Tone_1.DT_01(WFA_POTS_Dial_Tone_1.java:141)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- DT_01 -->
  <testcase name="DT_02" time="13.922" classname="TestCases.WFA_POTS_Dial_Tone_1"/>
  <testcase name="DT_03" time="13.809" classname="TestCases.WFA_POTS_Dial_Tone_1"/>
  <testcase name="DT_04" time="12.721" classname="TestCases.WFA_POTS_Dial_Tone_1"/>
  <testcase name="DT_05" time="14.122" classname="TestCases.WFA_POTS_Dial_Tone_1"/>
  <testcase name="DT_06" time="0.003" classname="TestCases.WFA_POTS_Dial_Tone_1"/>
  <testcase name="DT_23" time="16.43" classname="TestCases.WFA_POTS_Dial_Tone_2">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [106] line_end; expected [I would like to assist you in determining if the trouble is at your location or in your telephone equipment. Are you interested in hearing the trouble-shooting steps?If the customer is interested, select &amp;apos;Willing to check&amp;apos;] but found [I would like to assist you in determining if the trouble is atyour location or in your telephone equipment. Are youinterested in hearing the trouble-shooting steps?If the customer is interested, select &amp;apos;Willing to check&amp;apos;]gui_start  gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
@@line_start [106] line_end; expected [I would like to assist you in determining if the trouble is at your location or in your telephone equipment. Are you interested in hearing the trouble-shooting steps?If the customer is interested, select 'Willing to check'] but found [I would like to assist you in determining if the trouble is atyour location or in your telephone equipment. Are youinterested in hearing the trouble-shooting steps?If the customer is interested, select 'Willing to check']gui_start  gui_end
at TestCases.WFA_POTS_Dial_Tone_2.DT_23(WFA_POTS_Dial_Tone_2.java:141)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- DT_23 -->
  <testcase name="DT_24" time="0.003" classname="TestCases.WFA_POTS_Dial_Tone_2"/>
</testsuite> <!-- test_1 -->
